zerver/views/message_fetch.py

```diff
@@ -212,6 +233,7 @@ def get_messages_backend(
             include_anchor=include_anchor,
             num_before=num_before,
             num_after=num_after,
+            message_ids=message_ids_to_fetch,
```

Comment:
I'd prefer to keep the parameter name unchanged in this API boundary, to keep the code easier to trace.

Reply:
Okay, used `client_requested_message_ids`.